deployment/k8s/charts/aggregates/migrationAssistant/values.yaml

```diff
@@ -32,7 +32,7 @@ shared-configs:
         basicAuthPassword: "myStrongPassword123!"
         # Should be in sync with above username/password
         basicAuthHeader: "Basic YWRtaW46bXlTdHJvbmdQYXNzd29yZDEyMyE="
-        version: "2.16.0"
+        version: "OS_2.16"
```

Comment:
I'd prefer that these are more 'type-safe', can we switch to using an enum of something similar to prevent strange scenarios?

Reply:
While that makes sense it creates fragility in our system where it is non-obvious why something isn't working if these values are slightly off.  I hope that doesn't often occur.